### PR TITLE
fix: vim.loop.cpu_info()  report an error

### PR DESCRIPTION
**Pull request by HuiCheng**
I'm a termux user trying to fix a problem using neotext.

### CODE REVIEW

**Comment by rcarriga** [comment]
Can you provide more information as to why this change is needed?

**Comment by HuiCheng** [comment]
image

File: lua/neotest/client/strategies/init.lua [hunk 1]
```diff
@@ -13,9 +13,17 @@ end
 local ProcessTracker = {}
 
 function ProcessTracker:new()
+  local cpu_core = 4
+
+  if string.find(vim.loop.os_uname().release, "android") then
+    cpu_core = tonumber(vim.fn.system("nproc --all")) + cpu_core
```
Comment (rcarriga) [inline lua/neotest/client/strategies/init.lua:19]: `vim.fn.system` is blocking, please use `nio.process.run` instead

**Comment by rcarriga** [comment]
Closed in favour of #427

**Pull request closed without merging**